frame/store.cc

```diff
@@ -60,9 +60,10 @@ static inline uint64 HashBytes(const void *ptr, size_t len) {
 void Region::reserve(size_t bytes) {
   size_t used = size();
   DCHECK_LE(used, bytes);
-  base_ = static_cast<Address>(realloc(base_, bytes));
-  CHECK(base_ != nullptr || bytes == 0);
-  CHECK_EQ((reinterpret_cast<uintptr_t>(base_) & (kObjectAlign - 1)), 0);
+  Address tmp_ptr = static_cast<Address>(realloc(base_, bytes));
```

Comment:
According to realloc() definition:
"A pointer to the reallocated memory block, which may be either the same as ptr or a new location."
returned pointer could point to a new location, it is a good practice to assign it to a temporal variable to avoid undesired values in case the function fails.